### ISSUE TITLE
BetaFishIE: 8 image needs alt=""

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#312]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/betafish/

`<img id="EightWatermark" src="Images/Eight.svg?_ts=1483014193441">`

Should be:

`<img role="none" id="EightWatermark" src="Images/Eight.svg?_ts=1483014193441" alt='">`

Ideally would be a background image, but that might change perf in some way.

**Comment by alrra** [comment]
Closing this in favor of #319.
